### ISSUE TITLE
Update DOCS.md on location of apps source files

### ISSUE THREAD

**Issue by gmeisl** [hassio-addons/app-appdaemon#361]
# Problem/Motivation

It seems that with the AppDaemon Addon the source code for apps should be placed in /root/addon_configs/a0d7b954_appdaemon/apps and not under config/apps (as written in all AppDaemon documentation/tutorials that I found)

## Expected behavior

>  DOCS.md should state where application code should be located (and also other potential differences to https://appdaemon.readthedocs.io/en/latest/index.html )

## Proposed changes

> More explicit documentation. Thanks!

**Comment by DTekNOdev** [comment]
I have just struggled through this. It is not mentioned anywhere in the docs where the folder and configuration file is located on HAOS. It is implied that it should be in /config/apps and /config/appdaemon.html which is wrong

Installation and "hello world" takes about an hour more than necessary because of this!

**Comment by jarrah31** [comment]
I agree, I couldn't find AppDaemon.conf either because its location isn't mentioned in the documentation. 
To make it more accessible from the home assistant conference folder, I created the following symbolic link:
`ln -s /root/addon_configs/a0d7b954_appdaemon /root/homeassistant/appdaemon`

**Comment by gmeisl** [comment]
This obviously is an issue affecting more people and should be easy to fix.
